path.cabal

```diff
@@ -75,12 +75,12 @@ test-suite validity-test
                    , base       >= 4.7 && < 5
                    , bytestring
                    , filepath   < 1.2.0.1  || >= 1.3
-                   , genvalidity >= 0.3 && < 0.4
-                   , genvalidity-property >= 0.0 && < 0.1
+                   , genvalidity
+                   , genvalidity-property
                    , hspec      >= 2.0     && < 3
                    , mtl        >= 2.0     && < 3
                    , path
-                   , validity   >= 0.3.1.1 && < 0.4
+                   , validity
```

Comment:
Can you at least add lower bounds? I personally don't care about upper bounds for test-suite dependencies.